onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -434,6 +447,10 @@ void BasicBackend::StartAsyncInference(Ort::KernelContext& context, OVInferReque
       }
     }  // Loop subgraph original input names
 
+    if (!session_context_.shape.empty()) {
```

Comment:
This infer should not be here.

Reply:
Infer should only be called from Concrete backend or dynamic backend... this is breaking existing design